connectors/sources/tests/test_generic_database.py

```diff
@@ -155,8 +90,32 @@ def fetchmany(self, size):
         return []
 
     def fetchall(self):
```

Comment:
```suggestion
    def fetch_all(self):
```

Reply:
I see. Do we need a full class here or could we use mocking?

Reply:
As we have multiple functions in `Cursor` class as `keys`, `fetchmany` and `fetchall`. It's good to have full class.